functions/Get-DbaSpConfigure.ps1

```diff
@@ -55,55 +55,58 @@ function Get-DbaSpConfigure {
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
 		[Alias("ServerInstance", "SqlServer", "SqlServers")]
 		[DbaInstanceParameter[]]$SqlInstance,
-		[System.Management.Automation.PSCredential]$SqlCredential
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[object[]]$Config
```

Comment:
Go ahead and add `Silent` switch. Line 68 looks to be the only change required for `Write-Message`.